auth/jwt/transport.go

```diff
@@ -17,9 +17,9 @@ const (
 	bearerFormat string = "Bearer %s"
 )
 
-// ToHTTPContext moves JWT token from request header to context. Particularly
+// HTTPToContext moves a JWT from request header to context. Particularly
```

Comment:
Seems like you'd want to keep the "a" article use consistent—either use them everywhere or nowhere ("a request header", "a context").